media/videostitcher/stitcher_test.go

```diff
@@ -229,3 +228,169 @@ func getUUID64() (string, error) {
 	uuid := fmt.Sprintf("%s%s", u1.String(), u2.String())
 	return strings.ReplaceAll(uuid, "-", ""), nil
 }
+
+func createTestSlate(slateID string, t *testing.T) {
+	t.Helper()
+	ctx := context.Background()
+	client, err := stitcher.NewVideoStitcherClient(ctx)
+	if err != nil {
+		t.Fatalf("stitcher.NewVideoStitcherClient: %v", err)
+	}
+	defer client.Close()
+
+	tc := testutil.SystemTest(t)
+	req := &stitcherstreampb.CreateSlateRequest{
+		Parent:  fmt.Sprintf("projects/%s/locations/%s", tc.ProjectID, location),
+		SlateId: slateID,
+		Slate: &stitcherstreampb.Slate{
+			Uri: slateURI,
+		},
+	}
+	op, err := client.CreateSlate(ctx, req)
+	if err != nil {
+		t.Fatal(err)
+	}
+	_, err = op.Wait(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+}
+
+func deleteTestSlate(slateName string, t *testing.T) {
+	t.Helper()
+	ctx := context.Background()
+	client, err := stitcher.NewVideoStitcherClient(ctx)
+	if err != nil {
+		t.Errorf("stitcher.NewVideoStitcherClient: %v", err)
```

Comment:
I think this needs to be a `Fatalf`; otherwise L266 and L271 will seg fault. Or if you'd like code in the parent test to continue executing, you could do `Errorf` followed by return. Same elsewhere in these helper methods.

Reply:
Done